package.json

```diff
@@ -15,21 +15,22 @@
     "url": "git+https://github.com/doug-wade/i-want-to-be-a-fan.git"
   },
   "keywords": [],
-  "author": "",
-  "license": "ISC",
+  "author": "Doug Wade <douglas.b.wade@gmail.com>",
+  "license": "MIT",
   "bugs": {
     "url": "https://github.com/doug-wade/i-want-to-be-a-fan/issues"
   },
   "homepage": "https://github.com/doug-wade/i-want-to-be-a-fan#readme",
   "devDependencies": {
     "@11ty/eleventy": "^2.0.1",
-    "eslint": "^8.56.0",
-    "eslint-plugin-lit": "^1.11.0",
-    "eslint-plugin-wc": "^2.0.4"
+    "@tybalt/eleventy-plugin": "^0.1.6",
+    "@tybalt/eslint-plugin": "^0.1.6",
+    "@typescript-eslint/utils": "^6.18.1",
```

Comment:
We have to add this dependency, despite not using it, to work around https://github.com/doug-wade/tybalt/issues/586